src/style.css

```diff
@@ -1 +1,90 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(/src/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(/src/fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  --font-weight: 300;
+  --border-color: #e1e7ed;
+  --text-color: #3d4e61;
+}
+
+body {
+  margin-top: 0;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
Consider adding a fallback font for 'Avenir' to ensure consistent rendering across different browsers and systems. The current setup already includes Arial and sans-serif as fallbacks, which is good practice.